### ISSUE TITLE
Seq coordinates after trimming

### ISSUE THREAD

**Issue by Khalimat** [althonos/pytrimal#1]
Hi Martin, 

Thank you so much for making hmmer available in python and efficient (really did not expect this). I am using these packages (pyhymmer and pytrimal), and they are extremely useful!

A humble feature suggestion, I think it would be useful if it would be possible to record coordinates of trimmed sequences relative to full sequence ([sequence_id]/[start]-[end], as in GDF format for esl-sfetch with argument -C), as sometime they are useful. I can align trimmed seqs with initial ones and extract these coordinates, but it would be very inefficient option.

**Comment by Khalimat** [comment]
Sorry, I see that the question is irrelevant, as it could trim at any position of the sequence, not only at the starts and ends (so there is no single start/end coordinate). I would need to develop a trimming module for myself 😭😭😭.

**Comment by althonos** [comment]
>Thank you so much for making hmmer available in python and efficient (really did not expect this). I am using these packages (pyhymmer and pytrimal), and they are extremely useful!

Thanks, always happy to hear this!

>Sorry, I see that the question is irrelevant, as it could trim at any position of the sequence, not only at the starts and ends (so there is no single start/end coordinate)

Exactly, unless you use `TrimmedAlignment.terminal_only`, the trimming may happen inside the as well. Nevertheless, you can still extract a mapping of the new column coordinates to old column coordinates from the trimmed alignment residue mask:
```python
trimmed = trimmer.trim(alignment)
mapping = [ i for i, x in enumerate(trimmed.residues_mask) if x ] 
```

**Comment by Khalimat** [comment]
Thank you so much! I did not know about residues_mask attribute.

**Comment by althonos** [comment]
Looks like I didn't document it :( I will add some documentation for the next release.

**Comment by althonos** [comment]
Fixed in `v0.5.4`.
